app/app.go

```diff
@@ -982,7 +983,7 @@ func initParamsKeeper(appCodec codec.BinaryCodec, legacyAmino *codec.LegacyAmino
 	paramsKeeper.Subspace(ibctransfertypes.ModuleName)
 	paramsKeeper.Subspace(ibcexported.ModuleName)
 	// ethermint subspaces
-	paramsKeeper.Subspace(evmtypes.ModuleName)
+	paramsKeeper.Subspace(evmtypes.ModuleName).WithKeyTable(v0evmtypes.ParamKeyTable()) //nolint: staticcheck
```

Comment:
need to change evmKeeper.GetParams()?

Reply:
already [handled](https://github.com/crypto-org-chain/ethermint/blob/1b9176ea1e8479e631e500a07e12f29602f5374c/x/evm/keeper/params.go#L29)